### ISSUE TITLE
Glass pane connected textures not rendering properly

### ISSUE THREAD

**Issue by i0xHeX** [modmuss50/OptiFabric#201]
**Env:**
- Minecraft 1.15.2

**Mods:**
- Fabric API 0.7.1 b301 (1.15)
- OptiFabric 1.0.0-beta8
- OptiFine HD U G1 pre16 (also same with pre13)

**Details:**
Glass pane connected textures not rendering as intended if looking at `East` direction.
Setting `connected textures` set to `Fast` or `Fancy` gives no difference.

There are no such issues running single OptiFine without Fabric & OptiFabric.

![2020-05-13_13 43 42](https://user-images.githubusercontent.com/33582973/81803128-e95e0800-951f-11ea-8fb3-f9bf958488b5.png)
![2020-05-13_13 43 54](https://user-images.githubusercontent.com/33582973/81803141-ecf18f00-951f-11ea-80a9-6180caff9147.png)

**Comment by modmuss50** [comment]
Do you have the optifine internal resources enabled? in the resource pack window

**Comment by i0xHeX** [comment]
No

P.S. Enabled - now the glass pane connection is OK.

**Comment by i0xHeX** [comment]
Is that intended behavior or same should work without resources?

**Comment by grind651** [comment]
Yes, glass panels work correctly. But I have a problem with CTM Sanstone in the resource pack. Some sanstone blocks start showing default, with optifine internal resources enabled.
